modules/test/playwright/pages/commerce/commerceAdminChannelDetailsPage.ts

```diff
@@ -23,8 +32,8 @@ export class CommerceAdminChannelDetailsPage {
 		this.page = page;
 	}
 
-	async goto() {
-		await this.applicationsMenuPage.goToCommerceChannels();
+	async goto(checkTabVisibility = true) {
```

Comment:
I don't know if in this way is also optional (because otherwise you have to change ALL the other tests)